modules/client-configuration/nav-client-configuration-guide.adoc

```diff
@@ -27,11 +27,14 @@ endif::[]
 ** xref:gpg-keys.adoc[GPG Keys]
 ** xref:activation-keys.adoc[Activation Keys]
 ** xref:bootstrap-repository.adoc[Bootstrap Repository]
+// Contact Methods
 ** xref:contact-methods-intro.adoc[Client Contact Methods]
-*** xref:contact-methods-rhnsd.adoc[rhnsd]
-*** xref:contact-methods-pushssh.adoc[Push via SSH]
-*** xref:contact-methods-saltssh.adoc[Push via Salt SSH]
-*** xref:contact-methods-osad.adoc[OSAD]
+*** xref:contact-methods-salt.adoc[Contact Methods for Salt Clients]
```

Comment:
```suggestion
*** xref:contact-methods-salt.adoc[Management as Salt Clients]
```

Reply:
The preposition doesn't make sense here, it should be "of". How about starting with the verb instead: "Manage Salt Clients"?